README.md

```diff
@@ -20,3 +20,7 @@ services:
       - TOKEN=discord-secret
     restart: on-failure
 ```
+
+## Changelog
+
+0.2.0: Can parse game out of Steam Deck activity
```

Comment:
It'd probably make more sense to have a separate `CHANGELOG.md`, preferably using the [keep a changelog](https://keepachangelog.com/en/1.1.0/) format.